packages/by-name/contrast/package.nix

```diff
@@ -38,7 +46,13 @@ let
       "e2e/workloadsecret"
       "e2e/volumestatefulset"
       "e2e/regression"
+      "e2e/aks-runtime"
     ];
+
+    postInstall = ''
+      wrapProgram $out/bin/aks-runtime.test \
+        --prefix PATH : ${lib.makeBinPath [ azure-cli ]}
+    '';
```

Comment:
Now that we install the Azure CLI separately, we no longer need this, right? Furthermore, adding azure-cli to the PATH will make the test use that instead of the azure CLI that's already in PATH.